postreise/plot/plot_scatter_capacity_vs_capacity_factor.py

```diff
@@ -0,0 +1,96 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from postreise.analyze.generation.capacity import (
+    get_capacity_by_resources,
+    get_capacity_factor_time_series,
+)
+from postreise.analyze.helpers import get_plant_id_for_resources_in_area
+from postreise.analyze.time import change_time_zone, slice_time_series
+
+
+def plot_scatter_capacity_vs_capacity_factor(
+    scenario,
+    area,
+    resources,
+    time_zone="utc",
+    time_range=None,
+    area_type=None,
+    between_time=None,
+    dayofweek=None,
+    markersize=50,
+    fontsize=20,
+    title=None,
+    show_plot=True,
```

Comment:
do we want to add an optional parameter `use_percent` that would display the capacity factor in percent?

Reply:
Not sure how much value it will add given the capacity factor itself is a "percent" concept. There is no difference between values and percentage as we have elsewhere.

Reply:
I just thought about it because EIA uses % on their [plot](https://www.eia.gov/todayinenergy/detail.php?id=42995) and in their [table](https://www.eia.gov/electricity/monthly/epm_table_grapher.php?t=epmt_6_07_b). I personally have no preference

Reply:
I will add that then.

Reply:
Done.